.github/workflows/test.yml

```diff
@@ -325,6 +325,11 @@ jobs:
       - name: Install cargo rustfmt
         run: rustup component add rustfmt
 
+      - name: Install taplo
+        run: |
+          curl -LsSf https://github.com/tamasfe/taplo/releases/download/0.8.0/taplo-linux-x86_64.gz | gzip -d | sudo tee /usr/local/bin/taplo > /dev/null
```

Comment:
curious if we could use `npx @taplo/cli`?

Reply:
Yeah, that sounds good. That allows us to use the latest version without an unstable release url too.